apiaudio/api_request.py

```diff
@@ -114,18 +117,20 @@ def _get_request(cls, url=None, path_param=None, request_params=None):
     def _download_request(cls, url, destination, version=""):
         if type(url) is not str:
             raise TypeError("Error retrieving the audio files.")
-        
-        
-        remote_filename = url.split('/')[-1].split('?')[0]
+
+        remote_filename = url.split("/")[-1].split("?")[0]
         local_filename = f"{destination}/{remote_filename}"
         local_filename = local_filename.replace("%243ct10n", "section")
         local_filename = local_filename.replace("%7C", "|")
 
-        with requests.get(url, stream=True) as r:
-            cls._expanded_raise_for_status(r)
+        response = requests.get(url, stream=True)
+        while response.status_code == 202:  # keep the retrieval going if 202 returned
+            response = requests.get(url, stream=True)
+
+        cls._expanded_raise_for_status(response)
```

Comment:
here's the download fix to keep retrieving when 202 (in progress) is returned

Reply:
Do we not need a timeout here? Like a maximum retries?

Reply:
I don't think so